tensorflow/docs_src/deploy/s3.md

```diff
@@ -1,12 +1,57 @@
 # How to run TensorFlow on S3
 
-This document describes how to run TensorFlow on S3 file system.
+Tensorflow supports reading and writing data to S3. S3 is an object storage API which is nearly ubiquitious, and has many available implementations:
 
-## S3
+* [Amazon S3](https://aws.amazon.com/s3/)
+* [Google Storage](https://cloud.google.com/storage/docs/interoperability)
+* [Minio](https://www.minio.io/kubernetes.html)
+* [Ceph](http://docs.ceph.com/docs/master/radosgw/)
```

Comment:
S3 was tested against AWS before. It is interesting to see there are other implementations. Maybe the tests could be run against those implementations?

Reply:
I've manually tested against AWS, Google, and Minio (standalone, distributed has a bug). I haven't tested Ceph with TF but I'm told Radosgw does the job, and felt fair to represent it.

Or are you talking about automated tests?

Reply:
Thanks @elsonrodriguez. if S3 works on those systems I think we could keep it. Would you mind move these references to the end of the page? That way user will be easier to find the solution they need without reading through.

Reply:
Fixed, also removed Ceph since I didn't test it.

Reply:
Thanks for the update 👍 !